crates/red_knot_python_semantic/src/types.rs

```diff
@@ -321,7 +309,7 @@ impl<'db> Type<'db> {
     }
 
     #[must_use]
-    pub fn instance(&self) -> Type<'db> {
+    pub fn to_instance(&self) -> Type<'db> {
```

Comment:
I do like the rename from `instance` to `to_instance` or `as_instance` because it is a conversion method and not a simple getter.